### ISSUE TITLE
Some Required Corrections

### ISSUE THREAD

**Issue by maltintas45** [boun-tabi/UD_Turkish-BIMST#1]
Hi,

First of all, thanks for this edited version of a UD_Turkish-IMST treebank. I encountered some issues as below.

- In training set, following sentence; token 14 was written two times.

```# sent_id = 3160
# text = Sevdikçe, sevdiklerinden daha çok korkan insanların, başkalarına inanılmaz, tuhaf ve yabancı gözüken davranışlarını Sevda'ya anlatmam mümkün değildi.
1	Sevdikçe	sev	VERB	Verb	Aspect=Perf|Mood=Ind|Polarity=Pos|Tense=Pres|VerbForm=Conv	6	advcl	_	SpaceAfter=No
2	,	,	PUNCT	Punc	_	1	punct	_	_
3	sevdiklerinden	sev	VERB	Verb	Aspect=Perf|Case=Abl|Mood=Ind|Number[psor]=Sing|Person[psor]=3|Polarity=Pos|Tense=Past|VerbForm=Part	6	obj	_	_
4	daha	daha	ADV	Adverb	_	5	advmod	_	_
5	çok	çok	ADV	Adverb	_	6	advmod	_	_
6	korkan	kork	VERB	Verb	Aspect=Perf|Mood=Ind|Polarity=Pos|Tense=Pres|VerbForm=Part	7	acl	_	_
7	insanların	insan	NOUN	Noun	Case=Gen|Number=Plur|Person=3	16	nmod:poss	_	SpaceAfter=No
8	,	,	PUNCT	Punc	_	18	punct	_	_
9	başkalarına	başka	ADJ	NAdj	Case=Dat|Number=Plur|Number[psor]=Sing|Person=3|Person[psor]=3	15	obl	_	_
10	inanılmaz	inan	VERB	Verb	Aspect=Imp|Mood=Ind|Number=Sing|Person=3|Polarity=Neg|Tense=Aor|Voice=Pass	12	advmod	_	SpaceAfter=No
11	,	,	PUNCT	Punc	_	10	punct	_	_
12	tuhaf	tuhaf	NOUN	Noun	Case=Nom|Number=Sing|Person=3	15	amod	_	_
13	ve	ve	CCONJ	Conj	_	14	cc	_	_
14	yabancı	yabancı	ADJ	NAdj	Case=Nom|Number=Sing|Person=3	15	advmod	_	_
14	yabancı	yabancı	ADJ	NAdj	Case=Nom|Number=Sing|Person=3	12	conj	_	_
15	gözüken	gözük	VERB	Verb	Aspect=Perf|Mood=Ind|Polarity=Pos|Tense=Pres|VerbForm=Part	16	acl	_	_
16	davranışlarını	davranış	NOUN	Noun	Case=Acc|Number=Plur|Number[psor]=Plur|Person=3|Person[psor]=3	18	obj	_	_
17	Sevda'ya	Sevda	PROPN	Prop	Case=Dat|Number=Sing|Person=3	18	obl	_	_
18	anlatmam	anlat	VERB	Verb	Aspect=Perf|Case=Nom|Mood=Ind|Number[psor]=Sing|Person[psor]=1|Polarity=Pos|Tense=Pres|VerbForm=Vnoun	19	nsubj	_	_
19	mümkün	mümkün	ADJ	Adj	_	0	root	_	_
20	değildi	değil	VERB	Neg	Aspect=Perf|Mood=Ind|Number=Sing|Person=3|Polarity=Neg|Tense=Past	19	cop	_	SpaceAfter=No
21	.	.	PUNCT	Punc	_	19	punct	_	_
```
- In training set, following sentence; there is a cycle of heads.
```# sent_id = 3622
# text = Az kalsın kaçırıyordum ağzımdan.
1	Az	az	ADV	Adverb	_	2	advmod	_	_
2	kalsın	kal	VERB	Verb	Aspect=Perf|Mood=Imp|Number=Sing|Person=3|Polarity=Pos|Tense=Pres	1	compound	_	_
3	kaçırıyordum	kaçır	VERB	Verb	Aspect=Prog|Mood=Ind|Number=Sing|Person=1|Polarity=Pos|Polite=Infm|Tense=Past	0	root	_	_
4	ağzımdan	ağız	NOUN	Noun	Case=Abl|Number=Sing|Number[psor]=Sing|Person=3|Person[psor]=1	3	compound	_	SpaceAfter=No
5	.	.	PUNCT	Punc	_	3	punct	_	_
```
- Some POS tags and relations such as iobj, Ly etc does not exist in training set but exist in dev and/or test set, it may cause error prone cases.
